org.gaphor.Gaphor.json

```diff
@@ -44,7 +44,7 @@
       "build-commands": [
         "pip3 install --prefix=${FLATPAK_DEST} poetry",
         "${FLATPAK_DEST}/bin/poetry export --output=requirements.txt",
-        "pip3 install --prefix=${FLATPAK_DEST} -r requirements.txt"
+        "pip3 install --ignore-installed --prefix=${FLATPAK_DEST} -r requirements.txt"
```

Comment:
For my understanding, is ignore-installed overwriting something where force-reinstall couldn't be used?

Reply:
Force-reinstall tries to remove the old package. That does not work, since there is no list of files to remove.

Reply:
Got it, thanks!